### ISSUE TITLE
Symfony 2.8.3 and FOSBundle problem

### ISSUE THREAD

**Issue by iknowfoobar** [Kunstmaan/KunstmaanBundlesCMS#1049]
I haven't seen any issue about this on the FOSBundle site yet, however this morning I updated a 3.5 KunstmaanBundlesCMS website and the Symfony version went from 2.8.2 to 2.8.3. After the update I got the following error when trying to access the admin area:

```bash
Unable to find template "@FOSUserBundle/Resources/views/layout.html.twig" in FOSUserBundle:Security:login.html.twig at line 1.
```

Rolling back to 2.8.2 fixes this for me. I can also confirm that this is happening on a fresh install of v3.5 for me too.

OSX, PHP5.5.32 (homebrew)

I can confirm this also happens on Ubuntu 14.04, PHP5.5.9

**Comment by sandergo90** [comment]
Hi @iknowfoobar,

You have to use the dev-master for now. We have already fixed this and it will be released soon in a 3.5.1 tag.

Sander

**Comment by iknowfoobar** [comment]
3.5.1 fixes this.
